<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>632</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Ukraine"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="ResizableWidgetList" name="m_listChoice">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="locale">
       <locale language="Russian" country="Ukraine"/>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
      <property name="currentRow">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Data
input</string>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show the &lt;span style=&quot; font-weight:600;&quot;&gt;Data input&lt;/span&gt; panel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
       <property name="icon">
        <iconset resource="../images_resources.qrc">
         <normaloff>:/images/data_input.png</normaloff>:/images/data_input.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Queries</string>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show the &lt;span style=&quot; font-weight:600;&quot;&gt;Queries&lt;/span&gt; panel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
       <property name="icon">
        <iconset resource="../images_resources.qrc">
         <normaloff>:/images/queries.png</normaloff>:/images/queries.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Options</string>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show the application &lt;span style=&quot; font-weight:600;&quot;&gt;Options&lt;/span&gt; panel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
       <property name="icon">
        <iconset resource="../images_resources.qrc">
         <normaloff>:/images/options.png</normaloff>:/images/options.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEnabled</set>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="m_stackForms">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>450</height>
       </size>
      </property>
      <property name="locale">
       <locale language="Russian" country="Ukraine"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="m_actAboutApp"/>
    <addaction name="m_actAboutQt"/>
   </widget>
   <widget class="QMenu" name="m_menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="m_actEnglish"/>
    <addaction name="m_actUkrainian"/>
    <addaction name="m_actRussian"/>
   </widget>
   <widget class="QMenu" name="m_menuDatabase">
    <property name="locale">
     <locale language="Russian" country="Ukraine"/>
    </property>
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="m_actConnectToDB"/>
    <addaction name="m_actDisconnectFromDB"/>
   </widget>
   <widget class="QMenu" name="menuOperations">
    <property name="locale">
     <locale language="Russian" country="Ukraine"/>
    </property>
    <property name="title">
     <string>Operations</string>
    </property>
    <addaction name="m_actCreateEngine"/>
    <addaction name="m_actDeleteEngine"/>
    <addaction name="m_actSave"/>
    <addaction name="m_actRefresh"/>
    <addaction name="separator"/>
    <addaction name="m_actExit"/>
   </widget>
   <addaction name="menuOperations"/>
   <addaction name="m_menuDatabase"/>
   <addaction name="m_menuLanguage"/>
   <addaction name="m_menuHelp"/>
  </widget>
  <widget class="QToolBar" name="m_mainToolBar">
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_actCreateEngine"/>
   <addaction name="m_actDeleteEngine"/>
   <addaction name="m_actSave"/>
   <addaction name="m_actRefresh"/>
   <addaction name="separator"/>
   <addaction name="m_actConnectToDB"/>
   <addaction name="m_actDisconnectFromDB"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Debug output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="m_dbgOut">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>125</red>
             <green>53</green>
             <blue>31</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>125</red>
             <green>53</green>
             <blue>31</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>112</red>
             <green>111</green>
             <blue>113</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="m_actCreateEngine">
   <property name="icon">
    <iconset resource="../images_resources.qrc">
     <normaloff>:/images/add1.png</normaloff>:/images/add1.png</iconset>
   </property>
   <property name="text">
    <string>Create engine</string>
   </property>
   <property name="toolTip">
    <string>Create a new engine</string>
   </property>
  </action>
  <action name="m_actSave">
   <property name="icon">
    <iconset resource="../images_resources.qrc">
     <normaloff>:/images/save2.png</normaloff>:/images/save2.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save data in the database</string>
   </property>
  </action>
  <action name="m_actRefresh">
   <property name="icon">
    <iconset resource="../images_resources.qrc">
     <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh data in the application</string>
   </property>
  </action>
  <action name="m_actAboutApp">
   <property name="icon">
    <iconset resource="../images_resources.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About application</string>
   </property>
   <property name="toolTip">
    <string>Show information about this application</string>
   </property>
  </action>
  <action name="m_actAboutQt">
   <property name="icon">
    <iconset resource="../images_resources.qrc">
     <normaloff>:/images/qt-logo.png</normaloff>:/images/qt-logo.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>Show information about Qt</string>
   </property>
  </action>
  <action name="m_actEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="m_actUkrainian">
   <property name="text">
    <string>Ukrainian</string>
   </property>
  </action>
  <action name="m_actRussian">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="m_actConnectToDB">
   <property name="icon">
    <iconset resource="../images_resources.qrc">
     <normaloff>:/images/connection.png</normaloff>:/images/connection.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to the database</string>
   </property>
  </action>
  <action name="m_actDisconnectFromDB">
   <property name="icon">
    <iconset resource="../images_resources.qrc">
     <normaloff>:/images/disconnection.png</normaloff>:/images/disconnection.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from the database</string>
   </property>
  </action>
  <action name="m_actExit">
   <property name="icon">
    <iconset resource="../images_resources.qrc">
     <normaloff>:/images/exit2.png</normaloff>:/images/exit2.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit from the application</string>
   </property>
  </action>
  <action name="m_actDeleteEngine">
   <property name="icon">
    <iconset resource="../images_resources.qrc">
     <normaloff>:/images/clear_output.png</normaloff>:/images/clear_output.png</iconset>
   </property>
   <property name="text">
    <string>Delete engine</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete the current engine.&lt;/p&gt;&lt;p&gt;This action is enabled only if it was choosed Engines tab panel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ResizableWidgetList</class>
   <extends>QListWidget</extends>
   <header>widgets/reimplemented_widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images_resources.qrc"/>
 </resources>
 <connections/>
</ui>
